### DISCUSSION TITLE
离线安装有办法吗？

### DISCUSSION THREAD

**Discussion by kuo0528** [guangzhengli/kubenates_workshop#15] [General]
虽然网上好多资料，但往往都让我觉得，难道只有我没成功吗⋯⋯

**Comment by mohuani** [comment]
> 虽然网上好多资料，但往往都让我觉得，难道只有我没成功吗⋯⋯

你是说k8s环境的离线安装吗？你可以参看一下这篇文章，按照里面流程走下来，速度会快一点
- 文档：https://developer.aliyun.com/article/672675
